lib/nimble_template/addons/gettext.ex

```diff
@@ -0,0 +1,25 @@
+defmodule NimbleTemplate.Addons.Gettext do
+  @moduledoc false
+
+  use NimbleTemplate.Addon
+
+  @impl true
+  def do_apply(%Project{} = project, _opts) do
+    edit_mix(project)
+  end
+
+  defp edit_mix(%Project{mix_project?: true} = project) do
+    Generator.inject_content(
+      "mix.exs",
+      """
+        defp aliases do
+          [
+      """,
+      """
+        "gettext.extract-and-merge": ["gettext.extract --merge --no-fuzzy"],
```

Comment:
should we add `gettext.extract-and-merge` to the `codebase.fix` alias @byhbt 🤔

Reply:
Added in https://github.com/nimblehq/elixir-templates/pull/138/commits/eeb9256c0af1ff17907c8aee83b76a1e67a2ca6d